src/components/elements/ImageWithPlaceholder/ImageWithPlaceholder.tsx

```diff
@@ -27,7 +27,7 @@ const ImageWithPlaceholder: FC<ImageWithPlaceholderProps> = ({
     >
       <Image
         referrerPolicy="no-referrer"
-        src={hasErrored || !imageUrl ? noImageAvailable : imageUrl}
+        src={hasErrored || !imageUrl || !(imageUrl as StaticImageData).src ? noImageAvailable : imageUrl}
```

Comment:
Please use `src == null` and `imageUrl == null`. I think this could be simplified with:
```
src={hasErrored || (imageUrl as StaticImageData)?.src == null ? noImageAvailable : imageUrl}
```